pulser-pasqal/pulser_pasqal/backends.py

```diff
@@ -164,32 +165,53 @@ class EmuFreeBackend(PasqalEmulator):
     default_config = DEFAULT_CONFIG_EMU_FREE
 
 
-class EmuMPSBackend(RemoteBackend):
+class RemoteEmulatorBackend(RemoteBackend, EmulatorBackend):
+    _device_type: DeviceTypeName
+
     def __init__(
         self,
         sequence: pulser.Sequence,
         connection: PasqalCloud,
-        config: EmulationConfig,
+        *,
+        config: EmulationConfig | None = None,
         mimic_qpu: bool = False,
     ) -> None:
-        super().__init__(
+        RemoteBackend.__init__(
+            self,
             sequence=sequence,
             connection=connection,
             mimic_qpu=mimic_qpu,
         )
-        if not isinstance(config, EmulationConfig):
-            raise TypeError(
-                "'config' must be an instance of 'EmulationConfig', "
-                f"not {type(config)}."
-            )
-        self._config = config
-        self._device_type = pasqal_cloud.DeviceTypeName.EMU_MPS
+        EmulatorBackend.__init__(
+            self,
+            sequence,
+            config=config,
+            mimic_qpu=mimic_qpu,
+        )
```

Comment:
Let's replicate the line from https://github.com/pasqal-io/Pulser/pull/890 and mark it for removal once it's released.
```suggestion
        )
        self._config = type(self.default_config)(
            **{
                **self.default_config._backend_options,
                **config._backend_options,
            }
        )
```